frb_codegen/src/library/codegen/polisher/mod.rs

```diff
@@ -43,7 +43,7 @@ pub(super) fn polish(
     // Even if formatting generated code fails, it is not a big problem, and our codegen should not fail.
     if config.dart_format {
         warn_if_fail(
-            execute_dart_format(config, output_paths, progress_bar_pack),
+            execute_dart_format(config, progress_bar_pack),
```

Comment:
nit: it seems output_paths may not equal to "all files in folder", i.e. we may want to leave files that are not touched by us be unchanged.